### PR TITLE
Added Sesquicentennial footer bg image/svg text and styled

### PR DESCRIPTION
**Pull request by tomkrupka**
https://waynedotedu.tpondemand.com/entity/68220-program-the-sesquicentennial-footer

Looks like: https://app.hyfy.io/v/abBcD8b4f7n/

Still waiting on Design approval as well

### CODE REVIEW

**Review by nickdenardis — CHANGES_REQUESTED** [review]
This looks good! Can you do one pass through the Axe automated accessibility tool? Pretty sure the `<svg>` needs a `<title>` tag in it with the text of the SVG.

File: demo/index.html [hunk 1]
```diff
@@ -15,5 +15,12 @@
         </style>
     </head>
     <body>
-    </body>
+
+<div class="wsu-sesquicentennial__background">
+    <div class="wsu-sesquicentennial">
+        <a class="wsu-sesquicentennial__link" href="//150.wayne.edu">
```
Comment (nickdenardis) [inline demo/index.html]: This can go directly to `https://` since the 150 site forces SSL and some of the sites this is on may not.

File: src/footer.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+// Large
+@media (min-width: 600px) {
+    .wsu-sesquicentennial {
+        &__link {
+            padding: 30px;
+        }
+    }
+}
```
Comment (nickdenardis) [inline src/footer.scss]: This is small, but can you make sure your editor is picking up the `.editorconfig`? Per https://github.com/waynestate/sesquicentennial-footer/blob/feature/sesquicentennial-footer/.editorconfig#L10 it should be inserting a newline at the end of this file.

File: demo/index.html (hunk 1 above)
Comment (tomkrupka) [reply demo/index.html]: updated

File: src/footer.scss (hunk 2 above)
Comment (tomkrupka) [reply src/footer.scss]: added a new line

**Review by robertvrabel — APPROVED** [review]

**Comment by tomkrupka** [comment]
@nickdenardis I couldn't get axe extension to work locally.  But I did add the title tag in the svg.  Take a look plz

**Comment by nickdenardis** [comment]
@tomkrupka 
Got it. I'm running it now and will make the small change.

**Review by nickdenardis — APPROVED** [review]
This now passes WCAG 2.0 AA per aXe plugin.

**Comment by tomkrupka** [comment]
@nickdenardis I had already added inside the <g> following this example:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

Would you like me to remove mine then?
Nm I just removed it ;-)

**Comment by nickdenardis** [comment]
@tomkrupka Hmm.. Let me try again. It passed outside of the `<g>` tag.
